data.sql

```diff
@@ -11,3 +11,26 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Boarmon', '7/06/2005', 7, TRUE, 20.40);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Blossom', '13/10/1998', 3, TRUE, 17.00);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Ditto', '14/05/2022', 4, TRUE, 22.00);
+
+/* Insert details of animal owners */
+INSERT INTO owners(full_name, age) VALUES ('Sam Smith', 34);
+INSERT INTO owners(full_name, age) VALUES ('Jeniffer Orwell', 19);
+INSERT INTO owners(full_name, age) VALUES ('Bob', 45);
+INSERT INTO owners(full_name, age) VALUES ('Melody Pond', 77);
+INSERT INTO owners(full_name, age) VALUES ('Dean Winchester', 14);
+INSERT INTO owners(full_name, age) VALUES ('Jodie Whittaker', 38);
+
+/* Insert details of animal species */
+INSERT INTO species (name) VALUES ('Pokemon');
+INSERT INTO species (name) VALUES ('Digimon');
+
+/*Update animal's table */
+UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon';
+UPDATE animals SET owner_id = 2 WHERE name = 'Pikachu';
+UPDATE animals SET owner_id = 3 WHERE name = 'Devimon';
+UPDATE animals SET owner_id = 3 WHERE name = 'Plantmon';
+UPDATE animals SET owner_id = 4 WHERE name = 'Charmander';
+UPDATE animals SET owner_id = 4 WHERE name = 'Squirtle';
+UPDATE animals SET owner_id = 4 WHERE name = 'Blossom';
+UPDATE animals SET owner_id = 5 WHERE name = 'Angemon';
```

Comment:
- [OPTIONAL] Great job so far, but I think we can improve this a bit better 🤔

```sql
   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Jennifer Orwell')
   WHERE name = 'Gabumon' OR name = 'Pikachu';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Bob')
   WHERE name = 'Devimon' OR name = 'Plantmon';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Melody Pond')
   WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Dean Winchester')
   WHERE name = 'Angemon'  OR name = 'Boarmon';
```

Reply:
:+1: